doc/release-notes.rst

```diff
@@ -2,6 +2,123 @@
  Release Notes
 ===============
 
+v12.2.1 Luminous
+================
+
+This is the first bugfix release of Luminous v12.2.x long term stable
+release series. We recommend all the users of 12.2.x series update.
+
+Notable Changes
+---------------
+
+- *RGW*
+
+  * dynamic resharding is now enabled by default, RGW will now automatically
+  reshard the bucket index once the index grows beyond `rgw_max_objs_per_shard`
+
+- *CephFS*:
+
+  * Limiting MDS cache via a memory limit is now supported using the new
+    mds_cache_memory_limit config option (1GB by default).  A cache reservation
+    can also be specified using mds_cache_reservation as a percentage of the
+    limit (5% by default). Limits by inode count are still supported using
+    mds_cache_size. Setting mds_cache_size to 0 (the default) disables the
+    inode limit.
+
```

Comment:
Stylistically it's a little weird for the first two bullet points to be under headings (RGW, CephFS) but the last three not.

Also, headings are usually dispensed with when there is only one bullet point under them. Can you integrate the headings "RGW", "CephFS" into the text of the bullet points?